config/networkpolicy/allow-egress-kube-apiserver.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-egress-to-api-server
+spec:
+  podSelector:
+    matchLabels:
+      app: ipam-virt-workloads
+  policyTypes:
+  - Egress
+  egress:
+  - ports:
+    - protocol: TCP
+      port: 6443
```

Comment:
can't we be more specific than this ?

Reply:
Unfortunately we cant, network policy does not allow you to select endpoints on the host network such as kube-apiserver ones.